osu.Game.Rulesets.Osu/Skinning/Legacy/LegacyReverseArrow.cs

```diff
@@ -38,6 +40,10 @@ private void load(ISkinSource skinSource)
 
             InternalChild = arrow = (skin?.GetAnimation(lookupName, true, true, maxSize: OsuHitObject.OBJECT_DIMENSIONS) ?? Empty());
             textureIsDefaultSkin = skin is ISkinTransformer transformer && transformer.Skin is DefaultLegacySkin;
+
+            drawableObject.ApplyCustomUpdateState += updateStateTransforms;
```

Comment:
Kinda weird that there are event handlers being registered both in BDL and `LoadComplete()`.

Reply:
The animation doesn't play without it being here. We could try moving the other one here though?

Reply:
If it works, sure. If it doesn't, I guess I'll live with it.

Reply:
Honestly I'm not sure I'd feel safe changing it in this PR, so overlooking may be best 😅.